charts/teuto-portal/values.yaml

```diff
@@ -0,0 +1,50 @@
+global:
+  imageRegistry: registry-gitlab.teuto.net
+  imagePullSecrets: []
+  podSecurityContext:
+    runAsNonRoot: true
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    privileged: false
+    capabilities:
+      drop:
+        - ALL
+    runAsGroup: 1000
+    runAsUser: 1000
+
+  securityContext:
+    fsGroup: 1000
+    runAsUser: 1000
+    runAsGroup: 1000
+    runAsNonRoot: true
+    fsGroupChangePolicy: OnRootMismatch
+
+worker:
+  image:
+    repository: 4teuto/dev/teuto-portal/teuto-portal-k8s-worker/teuto-portal-k8s-worker
+    tag: 1.0.0
+  database:
+    user: root
+    password: ""
+    host: localhost
+    port: 5432
+  kubeconfig:
+    secret:
+      name: ""
+  config:
+    dryRun: false
+    singleRun: false
+    otelExporterTracesEndpoint: ""
+    syncInterval: 10s
+    watchK8sEvents: false
+    minimumCPUFlavour: 2
+    flavourPrefixFilter: ""
```

Comment:
This is a list of prefixes, in the environment variable joined by `, `

Reply:
so not a string instead something like this {[prefix1,prefix2,..] right?

Reply:
Yes, so by default `[]` and a test ci file with some entries

Reply:
done